### PR TITLE
Fixes ""word Blacklist""

### PR DESCRIPTION
**Pull request by Darker-Ink**

### CODE REVIEW

**Comment by Xcyth** [comment]
good work bruv

**Comment by wutv** [comment]
...but my pr ):

**Comment by Darker-Ink** [comment]
> ...but my pr ):

the point is this is fixing the code that was already there

**Comment by wutv** [comment]
image
won't work

**Comment by wutv** [comment]
you're checking if lowercase content includes uppercase content

**Comment by Darker-Ink** [comment]
> you're checking if lowercase content includes uppercase content

legit the reason its lowercase now

**Comment by Darker-Ink** [comment]
image

**Comment by Darker-Ink** [comment]
image

**Comment by Darker-Ink** [comment]
@freddierick can u merge pls and thanks

**Comment by freddierick** [comment]
This needs to be tested by someone before merge as there is quite a lot of changes.

**Comment by Darker-Ink** [comment]
I tried it on my bot, it worked fine.

File: bot/discord/events/message.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
 const fetch = require('node-fetch');
 const axios = require('axios');
 
-const blacklistedWords = [
-    'rape', 'nigga', 'nigger', 'jew'
-]
-
 module.exports = (client, message) => {
-    if (blacklistedWords.includes(message.content.toLowerCase())) { message.delete(), message.reply('Do __NOT__ use that word in this server. You will get muted next time...') }
```
Comment (FC5570) [inline bot/discord/events/message.js:9]: please return if the author is a bot @Darker-Ink

Comment (Darker-Ink) [reply bot/discord/events/message.js:9]: gonna kick the bot instead returning is a bad idea since bots could then just spam the nword

**Pull request merged**